### PR TITLE
[fix] Add --purge

### PR DESCRIPTION
**Pull request by Galettofraise**
## Problem

- *To add the option --purge*

## Solution

- *And how do you fix that problem*

## PR Status

- [x] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

File: scripts/remove [hunk 1]
```diff
@@ -76,6 +76,17 @@ ynh_script_progression --message="Removing app data directory..." --weight=1
 # Remove the app directory securely
 ynh_secure_remove --file="$data_path"
```
Comment (Jules-Bertholet) [inline scripts/remove:77]: You need to get rid of this line

**Review by Jules-Bertholet — CHANGES_REQUESTED** [review]

File: scripts/remove (hunk 1 above)
Comment (Galettofraise) [reply scripts/remove]: Ok i fix that

**Review by Jules-Bertholet — APPROVED** [review]

**Comment by Jules-Bertholet** [comment]
!testme

**Review by zamentur — APPROVED** [review]

**Review by Jules-Bertholet — APPROVED** [review]

**Comment by Jules-Bertholet** [comment]
!testme

**Comment by yalh76** [comment]
`data_path` is checked during install ... that's why adding purge option doesn't work

**Pull request merged**